jobs/bosh-dns-windows/spec

```diff
@@ -20,11 +20,14 @@ packages:
 
 properties:
   address:
-    description: "Address in which the DNS server will bind"
-    default: 169.254.0.2
+    description: "Address on which the DNS server will bind. Defaults to host IP address"
+    default: ""
```

Comment:
does this change the default for windows 2012? is that a problem?

Reply:
It changes the default for both windows2012R2 and windows2016 to the host IP. We verified that this setup worked for both

Reply:
chatting with @dpb587-pivotal i think it would be nicer to change the default to remain `nil` (as in no default value). in the erb templates you should be able to use `p("address", spec.ip)` (dont think to_s is needed anywhere).